From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Fri, 2 Oct 2026 09:59:52 +0300
Subject: [PATCH 008/192] kernel_build_check

Imported from https://github.com/openwrt/openwrt.git package/kernel/mac80211/patches/build/012-kernel_build_check.patch (added in d9eefa7a70); the patch file carries no author or description.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 754263c..989faff 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 # Makefile for the output source package
 #
 
-ifeq ($(KERNELRELEASE),)
+ifeq ($(KERNELVERSION),)
 
 MAKEFLAGS += --no-print-directory
 SHELL := /usr/bin/env bash
